doc/src/odin/index_markdown.md

```diff
@@ -0,0 +1,451 @@
+  |assign             |   forever        |  \*\*              |                   |  
+  |case               |   task           |  \^\~              |                   |  
+  |defparam           |   repeat         |  \<\<\<            |                   |  
+  |end                |  signed          | \>=                |                   | 
+  |endfunction        |  task            | \|\|               |                   | 
+  |endmodule          |   generate       |  \~&               |                   |  
+  |begin              |  genvar          | &&                 |                   | 
+  |default            |                  | \<\<               |                   | 
+  |else               |                  |\<=                 |                   |
+  |endcase            |                  |\>\>                |                   |
+  |endspecify         |                  | \~\^               |                   | 
+  |for                |                  | \~|                |                   | 
+  |function           |                  |\$clog()            |                   |
+  |if                 |                  |-:                  |                   |
+  |inout              |                  |+:                  |                   | 
+  |input              |                  |                    |                   |
+  |integer            |                  |                    |                   |
+  |localparam         |                  |                    |                   |
+  |module             |                  |                    |                   |
+  |nand               |                  |                    |                   |
+  |negedge            |                  |                    |                   |
+  |nor                |                  |                    |                   |
+  |not                |                  |                    |                   |
+  |or                 |                  |                    |                   |
+  |output             |                  |                    |                   |
+  |parameter          |                  |                    |                   |
+  |posedge            |                  |                    |                   | 
+  |reg                |                  |                    |                   | 
+  |specify            |                  |                    |                   |
+  |while              |                  |                    |                   |
+  |wire               |                  |                    |                   |
+  |xnor               |                  |                    |                   |
+  |xor                |                  |                    |                   |
+  |macromodule        |                  |                    |                   |
+  
+
+#### Verilog Syntax support:
```

Comment:
avoid punctiuation at the end of a header

Reply:
I put it through a converter to change from rtl to markdown, I had to manually do the tables and it screwed up the syntax for some things so I'll go through and check for little things like that

Reply:
cool! thanks for doing this :)
while you are at it then, I think 
>\`\`\` {.sourceCode .verilog}

can be replaced with just 

> \`\`\`Verilog

etc